sumologic/sumologic_cse_log_mapping_vendor_product.go

```diff
@@ -0,0 +1,45 @@
+package sumologic
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+)
+
+func (s *Client) GetCSELogMappingVendorsAndProducts(product, vendor string) (*CSELogMappingVendorAndProduct, error) {
+
+	data, _, err := s.Get(fmt.Sprintf("sec/v1/vendors-and-products?product=%s&vendor=%s", url.QueryEscape(product), url.QueryEscape(vendor)))
```

Comment:
I don't see this endpoint in API docs https://api.sumologic.com/docs/sec. Are we planning to release this API?

Reply:
currently is defined as private, but is already there ( being used in acceptance test  )